dev/com.ibm.ws.jaxrs.2.0_fat/publish/servers/com.ibm.ws.jaxrs.fat.simpleJson/server.xml

```diff
@@ -0,0 +1,9 @@
+<server>
+    <featureManager>
+        <feature>componenttest-1.0</feature>
+        <feature>jaxrs-2.0</feature>
+    </featureManager>
+    
+  	<include location="../fatTestPorts.xml"/>
+  	<javaPermission className="java.io.FilePermission" name="ALL FILES" actions="read"/>
```

Comment:
Why is this necessary?  Something required by Apache HTTP client?

Reply:
it was copied by an existing server, didn't investigate